docs/content/intro/readnext.md

```diff
@@ -22,6 +22,3 @@ Brigade's main documentation is broken up into “chunks” designed to fill dif
 The [introductory material][] is designed for people new to Brigade – or to web development on Kubernetes in general. It doesn’t cover anything in depth, but instead gives a high-level overview of how testing apps on Kubernetes “feels”.
 
 The [topic guides][topics], on the other hand, dive deep into individual parts of Brigade. There are complete guides to Brigade's controller, sandbox engine, and much more. This is probably where you’ll want to spend most of your time; if you work your way through these guides you should come out knowing pretty much everything there is to know about Brigade.
-
-[introductory material]: index.md
```

Comment:
I think these need to stay as long as they are used in-line as above (lines 22, 24)